src/main/resources/templates/organisation-api-keys.html

```diff
@@ -14,44 +14,49 @@
     <main class="govuk-main-wrapper govuk-main-wrapper--l" id="main-content" role="main">
         <div class="govuk-grid-row">
             <div class="govuk-grid-column-full">
-
-                <h1 class="govuk-heading-l">
-                    Manage API Keys
-                </h1>
-
                 <div class="govuk-grid-row">
-                    <div class="govuk-grid-column-one-third">
-                        <p class="govuk-body govuk-!-font-weight-bold">Department</p>
-                    </div>
+                    <div class="govuk-grid-column-two-thirds">
+                    <h1 class="govuk-heading-l">
+                        Manage API keys
+                    </h1>
 
+                    <dl class="govuk-summary-list govuk-summary-list--no-border">
+                        <div class="govuk-summary-list__row">
+                            <dt class="govuk-summary-list__key">
+                                Department
+                            </dt>
+<!--                            TODO : Replace with department from JWT-->
+                            <dd class="govuk-summary-list__value">
+                                Cabinet Office
```

Comment:
Can you update this as part of this MR? Should be able to get from the grant admin object, by either passing to the model, or grabbing directly in thymeleaf from session context

Reply:
(i prefer passing from model)